### DISCUSSION TITLE
How to compile GW on Fedora?

### DISCUSSION THREAD

**Discussion by romunov** [kcleal/gw#40] [Q&A]
Which packages and versions does one need to have installed (through `dnf`) in order to successfully compile GW on Fedora?

**Comment by kcleal** [comment]
Hi @romunov,

Good question! I have not tried on Fedora. Some of the main system packages include 

Glfw3
Curl
GLES
EGL
Htslib


However, the main issue is getting skia to work. The `make prep` in step GW will download a pre built lib, but as it was built on Debian it's not guaranteed to work. It may do though, so worth a try.

 If it doesn't work, then skia will have to be re built. I have a build script to help here. Let me know if the current lib works first.

**Comment by kcleal** [comment]
I forgot to mention, before installing htslib, it is well worth installing libdeflate, which will make file reading considerably quicker for most bam files!
